src/components/item_page/ItemPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './ItemPage.scss';
+import Page from 'components/UI/Page';
+import { countdown } from 'utilities/date';
+
+export const ItemPage = ({ item, error }) => {
+  let { id, title, description, price, end_date, image } = item;
+  return (
+    <Page>
+      <section className="error">{error}</section>
+      <section className="item--container" id={`item-${id}`}>
+        <img id={`item-image-${id}`} src={image} alt={title} />
+        <div className={`item-info`}>
+          <div className="item-title">
+            <span id={`item-title-${title}`}><strong>{title}</strong></span>
+          </div>
+          <div className="item-bidding-info">
+            <span id={`item-price-${price}`}>Current Bid: ${price}</span><br />
+            <span id={`item-end-date-${end_date}`}>{countdown(new Date(), new Date(end_date))}</span><br />
```

Comment:
`countdown` seems to always take "now" as the first argument, is that something worth pushing into the function? Or maybe in the form of another function like `countdownUntil(futureDate)`?

Reply:
We initially did that, but passing in the start date allowed us to test more directly. Is that a bad reason to make that design choice?

Reply:
Nah, driving your design using tests is great. In this case, I think having a wrapping function that can delegate would be a decent approach here.